src/vs/editor/contrib/referenceSearch/referenceSearch.ts

```diff
@@ -197,6 +212,16 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 	handler: closeActiveReferenceSearch
 });
 
+KeybindingsRegistry.registerCommandAndKeybindingRule({
+	id: 'openReferenceToSide',
+	weight: KeybindingsRegistry.WEIGHT.workbenchContrib(50),
+	primary: KeyMod.CtrlCmd | KeyCode.Enter,
+	mac: {
+		primary: KeyMod.WinCtrl | KeyCode.Enter
```

Comment:
Why isn't this `CtrlCmd`?

Reply:
I tried to be consistent with the other places where we have "Open to Side" (explorer, search, problems):

![image](https://user-images.githubusercontent.com/900690/34706578-f48b3054-f508-11e7-8066-2565ccb36554.png)